eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -13,49 +7,26 @@ steps:
       $VcpkgCommit = $(Get-Content eng/vcpkg-commit.txt)
       Write-Host "Target Path for vcpkg: $TargetPath"
       Write-Host "Vcpkg SHA: $VcpkgCommit"
+      Write-Host "Cache mode: $env:VCPKG_CACHE_MODE"
+      Write-Host "Cache SAS: $env:VCPKG_CACHE_SAS"
 
       Write-Host "##vso[task.prependpath]$TargetPath"
       Write-Host "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT]$TargetPath"
       Write-Host "##vso[task.setvariable variable=VcpkgCommit]$VcpkgCommit"
+      Write-Host "##vso[task.setvariable variable=VCPKG_BINARY_SOURCES;issecret=true]clear;x-azblob,https://cppvcpkgcache.blob.core.windows.net/public-vcpkg-container,$env:VCPKG_CACHE_SAS,$env:VCPKG_CACHE_MODE"
```

Comment:
@gearama , this is the source from where the pipeline would either read or readwrite